nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkComponent.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     pierre
+ */
+package org.nuxeo.ecm.core.bulk;
+
+import static java.util.Collections.singletonList;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+import org.nuxeo.ecm.core.api.NuxeoException;
+import org.nuxeo.runtime.model.ComponentContext;
+import org.nuxeo.runtime.model.ComponentInstance;
+import org.nuxeo.runtime.model.DefaultComponent;
+
+/**
+ * The bulk component.
+ *
+ * @since 10.2
+ */
+/*
+ * BulkAdminService is implemented by the component and not as a service like BulkService because StreamBulkScroller
```

Comment:
It's strange to have this non-Javadoc comment here, as it separates the Javadoc from the class declaration. Please move it above or inside the class.